src/ADAL.PCL/HttpClientWrapper.cs

```diff
@@ -115,8 +117,16 @@ public int TimeoutInMilliSeconds
                     {
                         requestMessage.Method = HttpMethod.Get;
                     }
-
+                    httpEvent.UserAgent=client.DefaultRequestHeaders.UserAgent.ToString();
+                    httpEvent.RequestApiVersion = requestMessage.Version.ToString();
+                    httpEvent.HttpQueryParameters = this.BodyParameters.ToString();
+                    httpEvent.HttpResponseMethod = requestMessage.Method.ToString();
+                    Telemetry.GetInstance().StartEvent(this.CallState.RequestId.ToString(),"http_time");
```

Comment:
>http_time [](start = 92, length = 9)

removed hardcoded strings. move it to a constants file. why is this called http_time?